File: messaging/pager-messages/create-internal-text-message/code-samples/createInternalTextMessage.py
# https://developers.ringcentral.com/my-account.html#/applications
# Find your credentials at the above url, set them as environment variables, or enter them below

# PATH PARAMETERS
accountId = '<ENTER VALUE>'
extensionId = '<ENTER VALUE>'

# POST BODY
body = {
    'from': {
        'extensionId': '<ENTER VALUE>'
    },
    'replyOn': 000,
    'text': '<ENTER VALUE>',
    'to': [
        {
            'extensionId': '<ENTER VALUE>'
        }
    ]
}

import os
from ringcentral import SDK
rcsdk = SDK(os.environ['clientId'], os.environ['clientSecret'], os.environ['serverURL'])
platform = rcsdk.platform()
platform.login(os.environ['username'], os.environ['extension'], os.environ['password'])
r = platform.post(f'/restapi/v1.0/account/{accountId}/extension/{extensionId}/company-pager', body)
# PROCESS RESPONSE

File: messaging/pager-messages/create-internal-text-message/code-samples/createInternalTextMessage.php
<?php
// https://developers.ringcentral.com/my-account.html#/applications
// Find your credentials at the above url, set them as environment variables, or enter them below

// PATH PARAMETERS
$accountId = '<ENTER VALUE>';
$extensionId = '<ENTER VALUE>';

// POST BODY
$body = array(
    'from' => array(
        'extensionId' => '<ENTER VALUE>'
    ),
    'replyOn' => 000,
    'text' => '<ENTER VALUE>',
    'to' => array(
        array(
            'extensionId' => '<ENTER VALUE>'
        )  
    )
);

require('vendor/autoload.php');
$rcsdk = new RingCentral\SDK\SDK(getenv('clientId'), getenv('clientSecret'), getenv('serverURL'));
$platform = $rcsdk->platform();
$platform->login(getenv('username'), getenv('extension'), getenv('password'));
$r = $platform->post("/restapi/v1.0/account/{$accountId}/extension/{$extensionId}/company-pager", $body);
// PROCESS RESPONSE
?>
